src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/types.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+export interface DataNormalizer {
+    normalize(data: SelectionData): SelectionData,
+}
+
+export type SelectionData = {
+    left: number,
+    top: number,
+    width: number,
+    height: number,
+};
+
+export type RectangleChange = {
+    topChange: number,
+    leftChange: number,
+    widthChange: number,
+    heightChange: number,
```

Comment:
I wouldn't really mind... I think it is quite common to share some variable names among different types. In that special case it's all of them (at least currently). And I am sure that the name of the variable holding these values will differ enough to justify omitting the `Change` here.